core-api/src/main/java/org/silverpeas/core/date/DateTime.java

```diff
@@ -35,7 +35,9 @@
 /**
  * A datetime, expressed in a given timezone. If no timezone is specified explicitly, then
  * the one of the JVM is used by default.
+ * @deprecated Use the java.time API
  */
+@Deprecated
 public class DateTime extends AbstractDateTemporal<DateTime> {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)